components/FlexibleHeader/src/MDCFlexibleHeaderView.h

```diff
@@ -64,6 +64,9 @@ IB_DESIGNABLE
  */
 @property(nonatomic, strong, nullable) CALayer *shadowLayer;
 
+/** The shadow color of the @c shadowLayer. */
+@property(nonatomic, copy, nonnull) UIColor *shadowColor;
```

Comment:
This is marked `nonnull` but I don't see a default value being set during initialization.  Please add the default color to common initializer method.

Reply:
Done